Rewrite Introduction + Abstract to remove focus on opt-out (#23)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

--- abstract

> This document proposes a standardized vocabulary for expressing preferences related to how digital assets are used by automated processing systems. 
> This vocabulary allows for the creation of structured declarations about restrictions or permissions for use of digital assets by such systems. 
> The vocabulary is agnostic to the means by which it is conveyed. 

Reviewer: ```suggestion
This document proposes a standardized vocabulary for expressing preferences related to how digital assets are used by automated processing systems.
This vocabulary allows for the creation of structured declarations about restrictions or permissions for use of digital assets by such systems.
The vocabulary is agnostic to the means by which it is conveyed.
```

The definitions in the vocabulary facilitate a shared understanding between entities that express such preferences and those that use the associated digital assets.

--- middle

# Introduction

This document defines a common vocabulary of terms for automated systems that process digital assets. The primary purpose of this vocabulary is to enable machine-readable expressions of preferences about how digital assets are used by automated processing systems, in the context of training AI models and other forms of text and data mining (TDM).

> The terms defined by the vocabulary can be used to describe, in a standardized way, the types of uses that a declaring party may wish to explicitly restrict or allow. This ensures that preferences—expressed as restrictions or permissions concerning specific types of use defined by the vocabulary—can be communicated, processed, and stored in a consistent and interoperable manner.

Reviewer: The middle clause of this second sentence seems a little rough to me.  Perhaps:

```suggestion
The terms defined by the vocabulary can be used to describe, in a standardized way, the types of uses that a declaring party may wish to explicitly restrict or allow.
Preferences are then expressed as a grant or denial of permission concerning each of the types of use defined in the vocabulary.
This ensures that preferences can be communicated, processed, and stored in a consistent and interoperable manner.
```

(I'm moving to sentence-per-line as much as possible so that these sorts of changes are easier to manage. The underlying format doesn't really care what we do, so I'm OK with anything really.)

> The vocabulary is agnostic to the technical implementations of how systems process preferences expressed via the terms defined by the vocabulary. It is designed to ensure that preference information can be effectively exchanged between different systems.

Reviewer: ```suggestion
The vocabulary is neutral to the technical details of how systems act on preferences.
It is designed to ensure that preference information can be exchanged between different systems and consistently understood.
```

> The vocabulary is intended to govern the use of digital assets in the context of training AI models and other forms of TDM, but does not concern itself with the mechanisms involved in obtaining digital assets (i.e. crawling).

Reviewer: ```suggestion
The vocabulary is intended to govern the use of digital assets for the training of AI models and other forms of automated processing.
It does not concern itself with the mechanisms involved in obtaining digital assets (i.e., crawling).
```

Trying to keep sentence length down, that's all.
